src/main/webapp/scroll-container.js

```diff
@@ -77,8 +77,12 @@ export class ScrollContainer extends HTMLDivElement {
    * If automatic scrolling is enabled or `forceScroll` is true, then scrolls
    * the container such that `element` is at the top of the container.
    * Otherwise, does nothing.
+   *
+   * <p>If `element` is undefined, the element to scroll is defaulted to the
+   * first element in the container storing the scrollable elements.
    */
-  scrollToTopOfContainer(element, forceScroll = false) {
+  scrollToTopOfContainer(
+      element = this.lastChild.firstChild, forceScroll = false) {
```

Comment:
`this.lastChild` refers to the container in ScrollContainer that store the scrollable elements (transcriptLineElements or comments)

Reply:
Please add what you wrote as a comment in the code.

Reply:
I updated the method header to mention this in the second paragraph.